selection_sort/python/selection_sort.py

```diff
@@ -0,0 +1,34 @@
+def selection_sort(arr): # Function that performs the SelectionSort
+    """
+    - The SelectionSort function takes in an array as argument
+    - Sorts the elements in array by making use of inner and outer loops
+    - Returns a sorted array as output
+    :papram arr: An array for sorting
+    """
+    n = len(arr) # Length of arr is stored for further use
+
+    for i in range(n): # Traverse through all array elements
+        low = i # Set the current element to low
+        for j in range(i+1, n): 
+            """
+            Traverse the array from i+1 to n-i as inner loop,
+            i.e. i is the counter for outer loop and j for 
+            the inner loop
+            """
+            if arr[j] < arr[low]: # Set low to current j, if it's value is smaller
+                low = j
+        arr[i], arr[low] = arr[low], arr[i] # Swap element at i with element low
+
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp7me9evv6/selection_sort/python/selection_sort.py
+++ b/tmp/tmp7me9evv6/selection_sort/python/selection_sort.py
@@ -12,7 +12,7 @@
         for j in range(i+1, n): 
             """
             Traverse the array from i+1 to n-i as inner loop,
-            i.e. i is the counter for outer loop and j for 
+            i.e. i is the counter for outer loop and j for
             the inner loop
             """
             if arr[j] < arr[low]: # Set low to current j, if it's value is smaller
```